src/CommitmentsV2/SFA.DAS.CommitmentsV2.Api/Controllers/ApprenticeshipsController.cs

```diff
@@ -25,18 +28,44 @@ public ApprenticeshipsController(IMediator mediator, ILogger<ApprenticeshipsCont
 
         [HttpGet]
         [Route("{providerId}")]
-        public async Task<IActionResult> GetApprenticeships(uint providerId)
+        public async Task<IActionResult> GetApprenticeships(uint providerId, [FromQuery]int pageNumber = 0, [FromQuery]int pageItemCount = 0, [FromQuery]string sortField = "", [FromQuery]bool reverseSort = false)
         {
             try
             {
-                var response = await _mediator.Send(new GetApprenticeshipsRequest {ProviderId = providerId});
+                var response = await _mediator.Send(new GetApprenticeshipsRequest
+                {
+                    ProviderId = providerId, 
+                    PageNumber = pageNumber, 
+                    PageItemCount = pageItemCount, 
+                    SortField = sortField,
+					ReverseSort = reverseSort
+                });
 
                 if (response == null)
                 {
                     return NotFound();
                 }
 
-                return Ok(response.Apprenticeships);
+                //TODO: Remove this mapping once we have consolidated the old Types with the new API types
```

Comment:
I wasn't sure what this means.

Reply:
I spoke to Dan about there being two type projects in commitments V2 (SFA.DAS.CommitmentsV2.Types and SFA.DAS.CommitmentsV2.Api.Types). From what I was told the API type project is the one moving forward and the general type project is for legacy support. The comment was about removing the mapping in the controller once we started to use the api type in the back-end once the general type project has been removed. Is this not the case?